Notebooks/README.md

```diff
@@ -7,4 +9,11 @@ downloaded from SSim meeting presentations. For this reason, you may
 sometimes have to modify the expected input file locations or obtain
 the needed input files etc. However, the information in these files
 will give you the background you need to write your own analysis and
-validation programs.
\ No newline at end of file
+validation programs.
+
+| Notebook | Short description | Links | Owner  |
+|----------|-------------------|-------|--------|
+| DC2 Postage Stamps | Make some small cutout images and visualize them | [ipynb](Notebooks/DC2%20Postage%20Stamps.ipynb), [html](http://htmlpreview.github.io/?https://github.com/LSSTDESC/DC2_Repo/blob/html/Notebooks/DC2_Postage_Stamps.html) | [Michael Wood-Vasey](https://github.com/LSSTDESC/DC2_Repo/issues/new?body=@wmwv) |
+| LSST Stack matching code | Spatial matching of objects using the DM Stack | [ipynb](Notebooks/LSST_Stack_matching_code.ipynb), [html](http://htmlpreview.github.io/?https://github.com/LSSTDESC/DC2_Repo/blob/html/Notebooks/LSST_Stack_matching_code.html) | [Jim Chiang](https://github.com/LSSTDESC/DC2_Repo/issues/new?body=@jchiang87)   |
```

Comment:
I did not attempt to order the notebooks in any way, I just added all three of Jim's into a table of all notebooks. The html links will point to rendered html pages, once they are made - but this will only happen once this branch is merged to master, and the beavis-ci script is run successfully...